api-reference/_hidden/GridBaseColumn/filterOperations.md

```diff
@@ -30,6 +30,8 @@ The following table lists available filters by [data types](/api-reference/_hidd
   </tr>
 </table>
 
+The nested filter builder also allows users to select from an extended set of operations that includes *"anyof"*, *"noneof"*, *"isblank"*, *"isnotblank"*, and [names](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations/#name) of custom operations (if there are any).
```

Comment:
```suggestion
The nested filter builder also allows users to select from an extended set of operations that include *"anyof"*, *"noneof"*, *"isblank"*, *"isnotblank"*, and [custom operations](/Documentation/ApiReference/UI_Widgets/dxFilterBuilder/Configuration/customOperations/#name).
```

Reply:
Hello. The point here is that the `filterOperations` array cannot contain custom operations, but only their names of the *string* type. The custom operations themselves should be implemented separately.

Reply:
I see. Ok, use [name] then